roborock/version_1_apis/roborock_client_v1.py

```diff
@@ -277,6 +277,8 @@ async def get_room_mapping(self) -> list[RoomMapping] | None:
         """Gets the mapping from segment id -> iot id. Only works on local api."""
         mapping: list = await self.send_command(RoborockCommand.GET_ROOM_MAPPING)
         if isinstance(mapping, list):
+            if not isinstance(mapping[0], list):
```

Comment:
Would you mind adding some length checks? I looks like mapping variable needs to be at least 2 items long

Reply:
done